src/assets/styles/main.scss

```diff
@@ -0,0 +1,10 @@
+@import 'reset.css';
+@import 'global.scss';
+
+/*
+** Main
+*/
+
+body {
+    font-family: 'Rubik';
```

Comment:
Will this now try to override the font on any website that imports this design system package and uses a component?

Reply:
Not quite sure yet but working on storybook, might need your help on this one